muted-tests.yml

```diff
@@ -402,4 +396,4 @@ tests:
 #    issue: "https://github.com/elastic/elasticsearch/..."
 #  - class: "org.elasticsearch.xpack.esql.**"
 #    method: "test {union_types.MultiIndexIpStringStatsInline *}"
-#    issue: "https://github.com/elastic/elasticsearch/..."
\ No newline at end of file
+#    issue: "https://github.com/elastic/elasticsearch/..."
```

Comment:
what is up with this line?

Reply:
My IDE added a newline at the end of the YAML file